server/rpm/pbench-server.spec.j2

```diff
@@ -25,12 +25,19 @@ Requires: npm
 
 Requires: python3-alembic python3-aniso8601 python3-boto3 python3-click
 Requires: python3-dateutil python3-elasticsearch python3-email-validator
-Requires: python3-flask python3-flask-cors python3-flask-httpauth
+Requires: python3-flask python3-flask-cors
+%if 0%{?rhel} != 8
+Requires:  python3-flask-httpauth
```

Comment:
I'm wondering if a comment to the effect that `# Not available on RHEL8` might avoid some future confusion? That is, explicitly recording that it's not because we don't need it but because it ain't there to need?